src/protocols/secure_channel/PairingSession.h

```diff
@@ -95,6 +96,12 @@ class DLL_EXPORT PairingSession
 
     CHIP_ERROR ActivateSecureSession(const Transport::PeerAddress & peerAddress);
 
+    void Finish();
+
+    void AbortExchange();
+    void CloseExchange();
+    void DiscardExchange();
```

Comment:
Please document the distinctions between these three methods.  DiscardExchange was documented in CASESession and PASESession but those comments got lost.  The distinction between AbortExchange and CloseExchange is non-obvious and should be documented.

In particular, CASESession and PASESession are not consistent in terms of whether they abort or close in `Clear()`.  I would think they should do the same thing, whichever thing that is.

Reply:
And specifically: maybe `PairingSession::Clear()` should do the `AbortExchange` and we remove the calls to that and DiscardExchange from the subclass clear methods.  Then the comment about Clear in Finish makes sense, and the behavior is consistent....

Reply:
`DiscardExchange` doesn't make sense if we use RAII for `PairingSession`, I'll keep current logic, and improve it when we decide to use RAII for it.